src/main/java/susan/task/Task.java

```diff
@@ -0,0 +1,34 @@
+package susan.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
```

Comment:
It's great that you use the function name to explain the purpose of the function